### PR TITLE
Fix position:sticky; prefix for Safari browser

### LINKED ISSUE #257: Position sticky is incorrectly prefixed
**Current behavior:**

```
prefix(`position:sticky;`, 8)
```
this code produces incorrect CSS:
```
-webkit-position:sticky;position:sticky;
```

**To reproduce:**

https://codesandbox.io/s/stilis-sticky-webkit-prefix-f6bro?file=/src/App.js:210-239

**Expected behavior:**

Generated CSS should look like this
```
position: -webkit-sticky; /* Safari */
position: sticky;
```
Source:
https://www.w3schools.com/howto/howto_css_sticky_element.asp

**Environment information:**

- `stylis` version: 4.0.7

### PR DESCRIPTION
**Pull request by isBatak**
Fix #257

Source: https://www.w3schools.com/howto/howto_css_sticky_element.asp

### CODE REVIEW

File: src/Prefixer.js [hunk 1]
```diff
@@ -84,8 +84,11 @@ export function prefix (value, length) {
 		// display: (flex|inline-flex|inline-box)
 		case 6444:
 			switch (charat(value, strlen(value) - 3 - (~indexof(value, '!important') && 10))) {
-				// stic(k)y, inline-b(o)x
-				case 107: case 111:
+				// stic(k)y
+				case 107:
+					return replace(value, 'sticky', WEBKIT + 'sticky') + value
```
Comment (Andarist) [inline src/Prefixer.js:89]: Could be written as:
```suggestion
					return replace(value, ':', ':' + WEBKIT) + value
```

**Comment by thysultan** [comment]
In that case `inline-box` should also have the same issue.

**Comment by Andarist** [comment]
@thysultan what is inline-box anyway? can't really find quickly anything about it (apart from maybe just this one: https://caniuse.com/mdn-css_properties_display_xul_box_values )

**Comment by thysultan** [comment]
@Andarist old flexbox version of `display: inline-flex`.

File: src/Prefixer.js (hunk 1 above)
Comment (thysultan) [reply src/Prefixer.js:89]: Yes this would handle both.

**Comment by Andarist** [comment]
@thysultan shouod we care about prefixing non-standard properties to non-standard properties though? 😅

**Comment by thysultan** [comment]
Yeah i guess not, shouldn't be there, the one below it handles `inline-flex` to `inline-box` already.

**Comment by Andarist** [comment]
@thysultan K, I've merged this one in and just pushed out to master a separate commit dropping `inline-box` prefixing. I hope that's OK

**Pull request merged**